index.html

```diff
@@ -47,5 +47,180 @@ <h2 class="h2_gray_title">Glad to see you!</h2>
+        <div class="card_img_container">
+          <img src="assets/cardImgs/Snapshoot Portfolio-2.png" alt="" />
+        </div>
+        <div class="card_description_container">
+          <h3>Tonic</h3>
+          <div class="card_projectType_container">
+            <p class="canopy">CANOPY</p>
+            <div>●</div>
+            <p class="project_type">Back End Dev</p>
+            <div>●</div>
+            <p class="project_year">2015</p>
+          </div>
+          <p class="card_description">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="card_languages">
+            <li>JavaScript</li>
+            <li>HTML</li>
+            <li>CSS</li>
+          </ul>
+        </div>
+        <button class="card_btn">See project</button>
+      </div>
+      <div class="cards">
+        <div class="card_img_container">
+          <img src="assets/cardImgs/Snapshoot Portfolio-3.png" alt="" />
+        </div>
+        <div class="card_description_container">
+          <h3>Multi-Post Stories</h3>
+          <div class="card_projectType_container">
+            <p class="canopy">CANOPY</p>
+            <div>●</div>
+            <p class="project_type">Back End Dev</p>
+            <div>●</div>
+            <p class="project_year">2015</p>
+          </div>
+          <p class="card_description">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="card_languages">
+            <li>JavaScript</li>
+            <li>HTML</li>
+            <li>CSS</li>
+          </ul>
+        </div>
+        <button class="card_btn">See project</button>
+      </div>
+    </section>
+
+    <section id="about_me_container">
+      <div id="about_me_img_container">
+        <img src="assets/aboutMeImg.png" alt="" />
+      </div>
+      <div id="about_me_description_container">
+        <h2 class="h2_gray_title">About myself</h2>
+        <p id="about_me_description">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website. Look through some of my work and experience! If
+          you like what you see and have a project you need coded, don’t
+          hesitate to contact me.
+        </p>
+        <p id="lets_connect">LET'S CONNECT</p>
+        <div id="socials">
+          <div>
+            <img src="assets/socials/twitter.svg" alt="Twitter icon" />
+          </div>
+          <div>
+            <img src="assets/socials/linkedin.svg" alt="Linkedin icon" />
+          </div>
+          <div>
+            <img src="assets/socials/m.svg" alt="m icon" />
+          </div>
+          <div>
+            <img src="assets/socials/github.svg" alt="Github icon" />
+          </div>
+          <div>
+            <img src="assets/socials/victory.svg" alt="victory icon" />
+          </div>
+        </div>
```

Comment:
- [ ] Great work building your social media list here 💯. However, please kindly ensure to use `<ul>` and `<li>` semantic tags instead of `<div>` and just `<img>` tags. 🤓 This will boost the readability and accessibility of your code and follows HTML best practices. 🔧